code/__DEFINES/dcs/signals.dm

```diff
@@ -290,8 +290,15 @@
 #define COMSIG_LIVING_START_PULL "living_start_pull"
 ///called on /living when someone is pulled (mob/living/puller)
 #define COMSIG_LIVING_GET_PULLED "living_start_pulled"
+///called on /living, when pull is attempted, but before it completes, from base of [/mob/living/start_pulling]: (atom/movable/thing, force)
+#define COMSIG_LIVING_TRY_PULL "living_try_pull"
+	#define COMSIG_LIVING_CANCEL_PULL (1 << 0)
```

Comment:
why are you doing this

Reply:
Players can right click mobs to get the pull verb, which dodged the components I set up to block clicking.

Reply:
I welcome suggestions to do this better, maybe making the pull verb go down the code path of ctrl click instead

Reply:
Further discussed in discord: issue was about indenting a define, not for having a pull signal
Answer: following precedent
Conclusion: precedent sucks

Happy to fix the whole file for you